recipe/meta.yaml

```diff
@@ -19,27 +21,35 @@ build:
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - pytorch 1.8                            # [build_platform != target_platform]
-    - pytorch-cpu                            # [build_platform != target_platform and osx or cuda_compiler_version in (undefined, 'None')]
-    - pytorch-gpu                            # [build_platform != target_platform and cuda_compiler_version not in (undefined, 'None')]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - {{ compiler('cuda') }}                 # [cuda_compiler_version not in (undefined, 'None')]
+    - sysroot_linux-64 ==2.17                # [linux64]
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - pytorch                                # [build_platform != target_platform]
+    - pytorch =*={{ torch_proc_type }}*      # [build_platform != target_platform]
   host:
     - python
     - pip
-    # pytorch 1.8 is the latest supported version of conda-forge at build time
-    - pytorch 1.8
-    - pytorch-gpu  # [cuda_compiler_version not in (undefined, 'None')]
-    - pytorch-cpu  # [osx or cuda_compiler_version in (undefined, 'None')]
+    - pytorch
```

Comment:
```suggestion
```

If we got the run_exports correctly in https://github.com/conda-forge/pytorch-cpu-feedstock/pull/89

you shouldn't need pytorch and it will pull in compatible versions only.

I would remove this to ensure that the tests pick up the right pytorch.